conda-forge.yml

```diff
@@ -1,3 +1,4 @@
+build_platform: {osx_arm64: osx_64, linux_ppc64le: linux_64, linux_aarch64: linux_64}
```

Comment:
Please use
```
build_platform:
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
  osx_arm64: osx_64
```
and sort it after `bot:`

Reply:
Done in https://github.com/conda-forge/mlir-python-bindings-feedstock/pull/32/commits/0f43913ef8c470741cfb52d776a4be7485986430.